src/Utility/Chebyshev.C

```diff
@@ -0,0 +1,233 @@
+// $Id$
+//==============================================================================
+//!
+//! \file Chebyshev.C
+//!
+//! \date Jul 13 2020
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Evaluation of Chebyshev polynomials.
+//!
+//==============================================================================
+
+#include "Chebyshev.h"
+#include "MatVec.h"
+
+#include <fstream>
+#include <numeric>
+
+
+double Chebyshev::evalPol1 (int polnum, double xi)
+{
+  if (polnum == 0)
+    return 1.0;
+  if (polnum == 1)
+    return xi;
+
+  return 2.0*xi*evalPol1(polnum-1, xi) - evalPol1(polnum-2, xi);
```

Comment:
If someone by error invokes evalPol1 with `polnum < 0` this will cause infinite recursion

Reply:
done.